### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#253] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152315578-c096c1a4-fde2-4085-bd82-312c048e4e8d.png)

**Comment by LuginiAndrea** [comment]
In modo completamente analogo all'esercizio 12, sappiamo che 
$$X = \left(X \cap \overline{Y}\right) \cup \left(X \cap Y\right) \Longrightarrow f\left(X\right) = f\left(X \cap \overline{Y}\right) \cup f\left(X \cap Y\right)$$
$$\Longrightarrow f\left(X\cap Y\right) \subset f\left(X \cap \overline{Y}\right) \cup f\left(X \cap Y\right) \cup f\left(Y\right)$$
Palese come ciò mostri la veridicità dell'ipotesi

**Comment by Elia-Belli** [comment]
Abbiamo già dimostrato che $f (X) \cup f(Y) = f(X \cup Y)$ per qualunque funzione f.

Siccome $X \cap Y \subseteq X \cup Y$ allora $f(X \cap Y) \subseteq f(X \cup Y) = f (X) \cup f(Y)$: 
questo perchè $f(X \cup Y)$ è l'immagine di tutti gli elementi presenti nell'unione di A e B, quindi è anche l'immagine degli elementi presenti nell'intersezione di A e B, siccome quest'ultima è contenuta nell'unione.
